desktop/wallet/main.js

```diff
@@ -31,11 +24,12 @@ function getBrowserWindowConfig() {
     center: true,
     show: false,
     webPreferences: {
-      devTools: isDevelopment,
-      nodeIntegration: true
+      devTools: context.isDevToolsEnabled(),
+      nodeIntegration: true,
+      contextIsolation: false,
```

Comment:
this little fucker is necessary to make ipcRenderer be accessible in renderer process.